hydra-core/lib/generators/hydra/head_generator.rb

```diff
@@ -36,6 +36,8 @@ def add_gems
     def inject_test_framework
       return if options[:'skip-rspec']
 
+      generate 'rspec:install'
+
       application("\n" <<
         "    config.generators do |g|\n" <<
```

Comment:
@jcoyne sorry, i'm not following what the requested changes are. or rather, what's missing from invoking the generate command on `rspec:install`.

when you say 'doesn't inject these lines' do you mean in your own testing the generate command didn't actually create the spec folder, the helpers, etc.? It did in mine, so just wanted to clarify.